packages/test/page-object/src/globals/pageConditions/scrollToBottomCompleted.js

```diff
@@ -0,0 +1,21 @@
+import getTranscriptScrollableElement from '../pageElements/transcriptScrollable';
+import stabilized from './stabilized';
+
+export default function scrollToBottomCompleted() {
+  return stabilized(
+    'scroll is at bottom and',
+    () => {
+      const scrollable = getTranscriptScrollableElement();
+      const { offsetHeight, scrollHeight, scrollTop } = scrollable;
+
+      // Browser may keep rendering content. Wait until consecutively:
+      // 1. Scroll position is at the bottom
+      // 2. Scroll top does not move
```

Comment:
```suggestion
      // Browser may keep rendering content. Wait until consecutively:
      // 1. Scroll position is at the bottom
      // 2. Scroll (to?) top does not move
```

Do you mean 'top of transcript' or something else?

Reply:
Discussed offline:

```js
      // Browser may keep rendering content. Wait until consecutively:
      // 1. Scroll position is at the bottom, and;
      // 2. Scroll is stabilized, i.e. `scrollTop` returns same value.
```